soh/soh/Enhancements/randomizer/randomizer_check_tracker.cpp

```diff
@@ -1044,7 +1044,7 @@ bool shouldHideArea(ImGuiTextFilter& checkSearch, std::map<RandomizerCheckArea,
         return false;
     }
     for (auto check : checksByArea[rcArea]) {
-        if (passesTextFilter(checkSearch, check)) {
+        if (IsVisibleInCheckTracker(check) && passesTextFilter(checkSearch, check)) {
```

Comment:
Oh ya know what, maybe we should put the visibility check in passesTextFilter... not that the visibility really matters for individual checks (just at the area level)